Taskfile.yml

```diff
@@ -0,0 +1,26 @@
+# https://taskfile.dev
+
+version: '3'
+
+vars:
+  GREETING: Hello, World!
+
+tasks:
+  default:
+    cmds:
+      - "{{.TASK_EXE}} --list"
+    silent: true
+  build:
+    desc: Build the image using nix.
+    cmds:
+      - nom build ".#hydraJobs.duos"
+  decompress:
+    desc: Decompress generated image to
+    cmds:
+      - mkdir -p output
+      - zstd -d result/sd-image/nixos-image-sd-card-25.05.20250328.2cc0d7f-riscv64-linux.img.zst -o output/nixos-image.img
```

Comment:
Try not to hardcode the version number?

Reply:
Yes, I can fix that. Or remove the whole Taskfile?

Reply:
Like, the taskfile is not really portable (e.g. /dev/sdc down there). Having it can cause more harm than not.